src/bidiMapper/utils/log.ts

```diff
@@ -15,16 +15,18 @@
  */
 export function log(type: string): (...message: any[]) => void {
   return (...messages: any[]) => {
-    const elementId = type + '_log';
+    console.log(type, ...messages);
```

Comment:
This (or some other `console.log`) is showing up in the output of `npm run unit`, I see this:
```
  CdpServer tests.
cdp sent > {"someAttribute":"someValue","id":0}
    ✔ given CdpServer, when `sendMessage` is called, then cdpBindings should be called with proper values
```

That "cdp sent" line would be nice to get rid of. If there is logging that's useful for debugging, then we should probably use a logging mechanism which can be disabled when running tests.

Reply:
done. Log only if run in browser.